cmd/notary-signer/main.go

```diff
@@ -191,7 +204,7 @@ func main() {
 		log.Println("HTTP server listening on", httpAddr)
 	}
 
-	err = server.ListenAndServeTLS(certFile, keyFile)
+	err = server.ListenAndServe()
```

Comment:
Addresses @mtrmac's [comment in #229](https://github.com/docker/notary/pull/229#discussion_r42788141)